tests/jballerina-unit-test/src/test/java/org/ballerinalang/test/statements/packageimport/PackageImportTest.java

```diff
@@ -73,15 +73,17 @@ public void testInvalidImport2() {
     @Test
     public void testImportsPerfile() {
         CompileResult result = BCompileUtil.compile("test-src/statements/package/sample_project_1");
-        Assert.assertEquals(result.getErrorCount(), 6);
+        Assert.assertEquals(result.getErrorCount(), 7);
         int i = 0;
         BAssertUtil.validateError(result, i++, "redeclared symbol 'int'", "file-negative1.bal", 3,
                 1);
         BAssertUtil.validateError(result, i++, "undefined module 'http'", "file-negative2.bal", 3, 5);
         BAssertUtil.validateError(result, i++, "unknown type 'Client'", "file-negative2.bal", 3, 5);
+        BAssertUtil.validateError(result, i++, "incompatible types: expected '()', found 'string'",
+                "file-negative2.bal", 3, 34);
         BAssertUtil.validateError(result, i++, "undefined function 'println'", "file-negative2.bal", 4, 5);
         BAssertUtil.validateError(result, i++, "undefined module 'io'", "file-negative2.bal", 4, 5);
-        BAssertUtil.validateError(result, i, "undefined module 'io'", "file-negative2.bal", 5, 18);
+        BAssertUtil.validateError(result, i++, "undefined module 'io'", "file-negative2.bal", 5, 18);
```

Comment:
No usage for post-increment value
```suggestion
        BAssertUtil.validateError(result, i, "undefined module 'io'", "file-negative2.bal", 5, 18);
```